libs/expo/betterangels/src/lib/screens/NotesHmis/HmisProgramNoteForm/formSchema.ts

```diff
@@ -1,60 +1,93 @@
 import { format } from 'date-fns';
 import { z } from 'zod';
+import { SelahTeamEnum, TaskStatusEnum } from '../../../apollo';
+
+// ----------------------------------------------------------------------
+// 1. Local Draft Task Type
+// ----------------------------------------------------------------------
+export type LocalDraftTask = {
+  id: string;
+  summary: string;
+  description?: string | null;
+  status?: TaskStatusEnum;
+  team?: SelahTeamEnum | null;
```

Comment:
```suggestion
```
same comment as above. I don't think it makes sense to have team on these tasks. Maybe if we had a generic team concept, but we just have a hardcoded list